apostrophecms/apostrophe
# conditional label whether doc is published or not

## Summary

Change the scheduled publishing field label, whether the doc is published or not (based on the doc's `lastPublishedAt` value).

## What are the specific steps to test this change?

- link apostrophe (on `pro-2745-conditional-label`) to testbed
- checkout the `scheduled-publishing` branch in testbed project and run it
- check that the label varies (see the screenshots below 👇)

**published** (doc already published)

<img width="1076" alt="image" src="https://user-images.githubusercontent.com/8301962/166930936-a97ecc90-8a48-456c-88d7-23d5ed14f062.png">

**new**  (doc not already published)

<img width="1072" alt="image" src="https://user-images.githubusercontent.com/8301962/166931022-537eaec8-ba88-442f-94ff-beab7ed9c7c4.png">

## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:
